TwoFAS/TwoFAS/Root/Modules/Settings/AppleWatch/View/AppleWatchView.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  This file is part of the 2FAS iOS app (https://github.com/twofas/2fas-ios)
+//  Copyright © 2024 Two Factor Authentication Service, Inc.
+//  Contributed by Grzegorz Machnio. All rights reserved.
+//
+//  This program is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  any later version.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program. If not, see <https://www.gnu.org/licenses/>
+//
+
+import SwiftUI
+import Common
+
+struct AppleWatchInstallationStep: Hashable, Identifiable {
+    var id = UUID()
+    let descirption: String
+    let actionTitle: String
+}
+
+struct AppleWatchView: View {
+    private let spacing: CGFloat = Theme.Metrics.doubleSpacing
+
+    let appleWatchInstallationSteps: [AppleWatchInstallationStep] 
+    let installationStepActionCallback: (_ stepNumber: Int) -> Void
+
+    var body: some View {
+        VStack(spacing: spacing) {
+            Image(uiImage: Asset.appleWatch.image)
+                .renderingMode(.original)
+                .padding(.top, Theme.Metrics.standardSpacing)
+
+            Text(T.AppleWatch.installationInfoTitle)
+                .font(Font(Theme.Fonts.Text.title))
+                    .multilineTextAlignment(.center)
+                    .minimumScaleFactor(0.7)
+
+            VStack(alignment: .leading) {
+                ForEach(Array(appleWatchInstallationSteps.enumerated()), id: \.element) { index, step in
```

Comment:
I would check that on smaller screens as a Scroll View can be necessary here

Reply:
![Screenshot 2024-07-22 at 23 01 41](https://github.com/user-attachments/assets/dc1b0131-aa82-447f-ad87-1f5ec5fa8839)

Reply:
IMO looks good on SE 3rd generation.